front-end/src/app/shared/services/report.service.ts

```diff
@@ -104,6 +104,10 @@ export class ReportService implements TableListService<Report> {
     return this.apiService.post(`${this.apiEndpoint}/${report.id}/amend/`, {});
   }
 
+  public startUnamendment(report: Report): Observable<string> {
```

Comment:
I see we're following the pattern the amendment implementation set.  What do you think of changing these two to promises? 

```
public amend(report:Report): Promise<string> {
  return getFirstValueFrom(this.apiService.post(...))
 }
```

Reply:
Oh I just saw this one. Yea promises would make a lot more sense here.